website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,138 @@
+
+### Custom content
+
+The `custom` type allows you to support any custom content, local components, or Helios components within the header via an instance swap property (customInstance) in Figma.
+
+![Generic container within the side-nav header](/assets/components/side-nav/custom-header.png)
+
+## Body
+
+The body consists of a group of sections with vertical lists of links, typically to the most important parts of the application. Any custom content or component is also supported by an additional generic container.
+
+### List
+
+#### With title
+
+![Side-nav section with a title](/assets/components/side-nav/section-with-title.png)
+
+#### Without title
+
+![Side-nav section without a title](/assets/components/side-nav/section-without-title.png)
+
+- A title could help users scan the sections and provide context about the links inside each section.
+
+- Titles should be meaningful and related to the content within the section.
+
+### List items
+
+#### Without icon
+
+![List items without icons](/assets/components/side-nav/list-item-without-icon.png)
+
+#### With icon
+
+![List items with icons](/assets/components/side-nav/list-item-with-icon.png)
+
+- Use icons to help users recognize and scan the links they are paired with.
+- We recommend only using icons in the main or top level navigation.
+- Avoid overwriting color styles in icons.
+
+!!! Do
+
+![List items with correct use of icons](/assets/components/side-nav/list-item-with-icon-do.png)
+!!!
+
+!!! Dont
+
+![List items with incorrect use of icons](/assets/components/side-nav/list-item-with-icon-dont.png)
+!!!
+
+#### With badge
+
+![List item with a badge](/assets/components/side-nav/list-item-with-badge.png)
+
+#### With count
+
+![List item with badge-count](/assets/components/side-nav/list-item-with-count.png)
+
+#### With nested items
+
+Set `hasSubItems` to show or signify that a link has a nested level of navigation.
+
+![List item with sub-items](/assets/components/side-nav/list-item-with-nested-items.png)
+
+#### External links
+
+Use `isLinkExternal` to show that the list item is a hyperlink pointing to a page outside the product or platform.
+
+![List item with a external link](/assets/components/side-nav/list-item-with-external-link.png)
+
+!!! Warning
+
+Use external links sparingly. Avoid using this property to link pages that are unrelated to the product's navigation.
+!!!
+
+### Custom content
+
+Toggle `hasCustomContent` on to support any additional custom content, local components, or Helios components within the body container via an instance swap property (customInstance) in Figma.
+
+![Generic container within the side-nav body](/assets/components/side-nav/custom-content-body.png)
+
+## Footer
+
+### Context switcher
+
+The context switcher allows users to switch between different contexts within your product or application. For example, the side navigation can display different links and content based on a particular organization or workspace the user selects.
```

Comment:
```suggestion
The context switcher allows users to switch between different contexts within your product or application. For example, the SideNav can display different links and content based on a particular organization or workspace the user selects.
```
Should this specifically reference the component? Open question, not opinionated

Reply:
@jorytindall I don't think it necessarily should, especially since it's an example to illustrate the purpose and intent of the context switcher. Would something like this sound better?

"The context switcher allows users to switch between different contexts within your product or application. For example, the navigation or application content can change based on a particular organization or workspace the user selects."

Reply:
Yep, I think that reads well, thanks for the clarification!